frontend/src/e2e-playwright/specs/4_finance/value-decisions.spec.ts

```diff
@@ -119,6 +119,13 @@ describe('Value decisions', () => {
     await valueDecisionsPage.navigateBackFromDetails()
   })
 
+  test('Send value decision from details page', async () => {
+    await valueDecisionsPage.openFirstValueDecision()
+    await valueDecisionsPage.sendValueDecision()
+    await valueDecisionsPage.navigateBackFromDetails()
+    await valueDecisionsPage.assertSentDecisionsCount(1)
+  })
```

Comment:
Seems like this test is flaky. You should wait for async jobs to complete at some point.